src/app/layouts/carousel/carousel-sample-1/carousel-sample-1.component.scss

```diff
@@ -1,7 +1,16 @@
-.igx-carousel__inner {
-    width: 100%;
-    text-align: center; 
-}
-igx-carousel igx-icon.igx-icon{
-    color: #e41c77;
+
+:host ::ng-deep{
+
+    .igx-carousel__inner {
+        width: 100%;
+        text-align: center; 
+    }
+
+    igx-carousel igx-icon.igx-icon{
```

Comment:
The `igx-icon.igx-icon` selector is a bit much. You can target the `igx-icon` only, as the igx-icon class is always set anyways.